src/rust/engine/fs/store/src/lib.rs

```diff
@@ -1826,7 +1916,13 @@ mod remote {
       call_option
     }
 
-    pub fn store_bytes(&self, bytes: Bytes) -> BoxFuture<Digest, String> {
+    pub fn store_bytes(
+      &self,
+      bytes: Bytes,
+      workunit_store: WorkUnitStore,
```

Comment:
We should do it in a separate change, but I continue to think that we should make the workunit store "task local": cc @illicitonion: it's only going to become more common to need to pass it around.